### PR TITLE
Better error message and docstring

### PR DESCRIPTION
**Pull request by nouiz**
fix gh-3665

I think this PR need an update to the error message to point to abstract_conv2d.

But I think we can merge now, people will search for it until this get done.

### CODE REVIEW

**Comment by lamblin** [comment]
If pep8 is fixed, can you remove the file from the whitelist?

**Comment by nouiz** [comment]
It wasn't in the whitelist. I fixed pep8 that the test tolerate like line too long.

**Comment by nouiz** [comment]
docgen.py --test

git rm doc/crei2013/logreg_profile.txt
            extraopts = ['-W']

array(188.4...) # doctest: ELLIPSIS

>>> numpy.allclose(f(94.2), 188.4)
True

**Comment by nouiz** [comment]
I updated this PR and I think it can be merged after review. I don't expect confict with other change.

File: doc/library/tensor/basic.txt [hunk 1]
```diff
@@ -510,7 +510,7 @@ TensorVariable
     .. method:: diagonal(offset=0, axis1=0, axis2=1)
     .. method:: astype(dtype)
     .. method:: take(indices, axis=None, mode='raise')
-    .. method:: copy()
+    .. method:: copy() Return an new symbolic variable that copy the variable. Do not copy the tag.
```
Comment (abergeron) [inline doc/library/tensor/basic.txt]: Do not copy -> Does not copy

Comment (abergeron) [inline doc/library/tensor/basic.txt]: Return an new -> Return a new
that copy -> that is a copy of

File: theano/tensor/var.py [hunk 2]
```diff
@@ -535,7 +535,10 @@ def take(self, indices, axis=None, mode='raise'):
 
     # COPYING
     def copy(self, name=None):
-        """Copy a variable and optionally assign a name."""
+        """Return a symbolic copy and optionally assign a name.
+
+        Do not copy the tags.
```
Comment (abergeron) [inline theano/tensor/var.py]: Do not copy -> Does not copy

**Comment by abergeron** [comment]
Looks good to merge except for a few typos.

**Comment by nouiz** [comment]
I updated the PR.

**Pull request merged**